core/panic-handler/src/lib.rs

```diff
@@ -31,7 +31,18 @@ use std::cell::Cell;
 use std::thread;
 
 thread_local! {
-	static ABORT: Cell<bool> = Cell::new(true);
+	static ON_PANIC: Cell<OnPanic> = Cell::new(OnPanic::Abort);
+}
+
+/// Panic action.
+#[derive(Debug, Clone, Copy, PartialEq)]
+enum OnPanic {
+	/// Abort when panic occurs.
+	Abort,
+	/// Unwind when panic occurs.
+	Unwind,
+	/// Always unwind even if someone changes strategy to Abort afterwards.
+	NeverAbort,
```

Comment:
Oh no, this is worrysome because if I see `force_abort` it doesn't mean that it will abort. The name `force_abort` doesn't add clarity as well...

Reply:
Yes - I agree :) I could rename to `unwind()`, `abort()` and `force_unwind()` if you think it is better. Or you could suggest some other naming